apps/condo/domains/ticket/utils/serverSchema/TicketChange.js

```diff
@@ -0,0 +1,153 @@
+        const item = await getById('User', itemId)
+        return _.get(item, 'name')
+    },
+    'assignee': async (itemId) => {
+        const item = await getById('User', itemId)
+        return _.get(item, 'name')
+    },
+    'executor': async (itemId) => {
+        const item = await getById('User', itemId)
+        return _.get(item, 'name')
+    },
+    'classifier': async (itemId) => {
+        const item = await getById('TicketClassifier', itemId)
+        return _.get(item, 'name')
+    },
+    'source': async (itemId) => {
+        const item = await getById('TicketSource', itemId)
+        return _.get(item, 'name')
+    },
+    'related': async (itemId) => {
+        const item = await getById('Ticket', itemId)
+        if (!item) {
+            return null
+        } else {
+            const number = _.get(item, 'number')
+            return number ? number.toString() : null
+        }
+    },
+}
+
+const relatedManyToManyResolvers = {
+    'watchers': async ({ context, existingItem, originalInput }) => {
+        let updated
+        const updatedResult = await context.executeGraphQL({
+            context: context.createContext({ skipAccessControl: true }),
+            query: `
+                query changeTrackable_findTicketWatchersForTicketChange($id: ID!) {
+                    ticket: Ticket(where: { id: $id }) {
+                        id
+                        watchers {
+                            id
+                            name
+                        }
+                    }
+                }
+            `,
+            variables: { id: existingItem.id },
+        })
+        if (updatedResult.errors) {
+            console.error('Error while fetching related items in relatedManyToManyResolvers of changeTrackable for a Ticket', updatedResult.errors)
+            return {}
+        }
+        if (updatedResult.data.ticket) {
+            updated = {
+                ids: _.map(updatedResult.data.ticket.watchers, 'id'),
+                displayNames: _.map(updatedResult.data.ticket.watchers, 'name'),
+            }
+        }
+
+        /*
+            Restore previous list of related items
+            This is a tricky part, because we need to restore previous list of related items,
+            that explicitly is not presented anywhere.
+            Following solutions was considered:
+            1. Get it from `beforeChange` hook of the Ticket, but where to store it?
+            2. Get previous TicketChange and get it from `watchersIdsTo` field.
+               This solution is not suitable for handling first change,
+               because there will be no `TicketChange` item yet.
+            3. Get current `watchers` list (after update) and make a replay,
+               based on Keystone `originalInput`, which stores "Nested mutation" operations
+               for this relationship. This variant is implemented.
+        */
+        let existing = { ...updated } // make a copy
```

Comment:
Use cloneDeep from lodash, default spread operator make shallow copy.

Reply:
Used in `669c4f5e`.